### ISSUE TITLE
Failing CI due to links to external package

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#8122]
## Provide a description of the task

In #8091 we added a couple of links that started to break the CI:

```
Error: Failed while convert {@link features/export-pdf exported to PDF} tag in ckeditor5/22.0.0/features/page-break.md.
Error: Failed while convert {@link features/export-word to Word} tag in ckeditor5/22.0.0/features/page-break.md.
Error: Failed while convert {@link features/pagination pagination feature} tag in ckeditor5/22.0.0/features/page-break.md.
Error: Failed while convert {@link features/pagination pagination feature} tag in ckeditor5/22.0.0/features/page-break.md.
Error: Failed while convert {@link features/export-pdf exported to PDF} tag in ckeditor5/22.0.0/features/page-break.md.
Error: Failed while convert {@link features/export-word to Word} tag in ckeditor5/22.0.0/features/page-break.md.
Error: Failed while convert {@link features/export-word export to Word} tag in ckeditor5/22.0.0/features/page-break.md.
Error: Failed while convert {@link features/export-pdf export to PDF} tag in ckeditor5/22.0.0/features/page-break.md.
```

See: https://travis-ci.org/github/ckeditor/ckeditor5/jobs/729221776

I have fixed a case like this a couple of days ago in #8061.

**Comment by godai78** [comment]
These are internal CKE5 features, we should not link them with absolute links... The structure looks correct :|

**Comment by AnnaTomanek** [comment]
@mlewand This is absolutely not an issue that should be fixed in the documentation by changing _valid_ links created as intended in the docs project, according to all the rules, and you should not workaround it by replacing links generated by our own docs building tools to a hardcoded URL because it kinda defeats the purpose of our own link validation mechanisms. If anything, this issue is a problem in your CI integration and something to be fixed there.

Please revert #8061 as this is not the proper way to fix this problem.

**Comment by jodator** [comment]
Hopefully, changes in CI would help with this. I'm leaning towards not validating external links in CKE5 CI and have a dedicated job for full documentation checking.

For now, improper use of an external link is a way to go. We rely on CI checks in PR reviews and I'd go with a not-ideal solution until we fix the CI.

TL;DR - we should fix CI then fix links.

**Comment by godai78** [comment]
These links built OK in our local versions, so they _are_ correct and should not result in any errors in the final render. I'd also stick to what You suggest.

**Comment by mlewand** [comment]
We discussed this matter on devops sync today and the only solution to get around this is to either to hook up a second CI that will have access to all the project (and remove the docs check from \`ckeditor/ckeditor5\` ci) or to revise entire Umberto project and rethink how we should handle cases like that.

Both solutions require substantial amount of effort that we can't afford at this point, so decided to stick to working around this particular annoyance.

**Comment by Reinmar** [comment]
I pushed a PR with a workaround: #8134. @godai78 could you check whether I didn't make a stupid mistake somewhere?

I'll explain why a workaround seems to be the only viable solution at the moment later on.

**Comment by Reinmar** [comment]
As for more context – in #8134 I used the same approach that we already used when linking to RTC docs: https://github.com/ckeditor/ckeditor5/blob/f0b5f2d/docs/builds/guides/frameworks/angular.md#L133.

Unfortunately, cross-repo links never worked and we have 3 options:

*   Use absolute URLs. This is what we did so far as it was the simplest and fastest solution. 
*   We can improve the linking system in Umberto to allow distinguishing between links within the current project and between projects. This way we'll be able to link to CF and other commercial features. Umberto may have a flag implemented to check either only the local links (CKE5's CI and local usage) or all links (integration CI). However, none of us really know how the linking system works now in Umberto (:facepalm: ) and touching it is not a quick job. It's a high risk of running into a big refactoring and spending 5+ days there.
*   We can stop checking docs on the CKE5's CI and move docs validation to the integration CI. However, this means that integration CI will have to run for all branches, not only for master. And this is again a quite significant effort as there are details such as security that we'd need to consider. Moreover, this is not really a solution but rather a workaround. Locally, you may still end up with these errors as you may not have the private repos installed.

Therefore, absolute URLs, until they really are causing us to lose a lot of time, seem the best option. In the meantime, though, we need to think about putting a bigger effort into Umberto to straighten this up there.

**Comment by godai78** [comment]
Absolute links open in new window/tab which is basically not good.
